snap/snapcraft.yaml

```diff
@@ -34,7 +34,7 @@ layout:
 plugs:
   wayland:
   opengl:
-  pulseaudio:
```

Comment:
You probably should not remove this just yet, in case someone gets this on older snapd? That's what the recommendation is anyway…